src/main/java/uk/gov/mca/beacons/service/dto/WrapperDTO.java

```diff
@@ -9,7 +9,7 @@
 
   private final Map<String, Object> meta = new HashMap<>();
   private T data = null;
-  private final List<DomainDTO> included = new ArrayList<>();
+  private final List<DomainDTO<?>> included = new ArrayList<>();
```

Comment:
This is new and exciting, what's the ? doing?

Reply:
It's called a [wildcard](https://docs.oracle.com/javase/tutorial/java/generics/wildcards.html) and [example](https://docs.oracle.com/javase/tutorial/extra/generics/wildcards.html) which essentially means the type of T is unknown.  

We don't know the type of T which is not going to be fixed as it could be of any type for entries within the list. The types that we're likely to use are are the DTO Pojos so:

- BeaconDTO
- AccountHolderDTO
- BeaconUseDTO

But in the list itself we could have `[DomainDTO<BeaconDTO>, DomainDTO<BeaconUseDTO>]` etc... which is why we've specified it as an unknown type